src/coreComponents/physicsSolvers/multiphysics/poromechanicsKernels/SinglePhasePoromechanics_impl.hpp

```diff
@@ -85,11 +85,13 @@ smallStrainUpdate( localIndex const k,
   real64 dPorosity_dPressure = 0.0;
   real64 dPorosity_dTemperature = 0.0;
   real64 dSolidDensity_dPressure = 0.0;
+  real64 timeIncrement = 0.0;
```

Comment:
Here and in the other mechanics/poromechanics kernels, should you pass the actual `dt` to have a complete implementation? Currently the `timeIncrement` are all hard-coded to 0.

Reply:
This will be fixed in the following PR, when working on boundary value problems.